apps/total-typescript/src/pages/youtube/[id].tsx

```diff
@@ -20,8 +29,47 @@ const YouTube = () => {
           title="Embedded youtube"
         />
       </div>
+      <Banner />
     </Layout>
   )
 }
 
 export default YouTube
+
+const Banner = () => {
+  return (
+    <div className="sm:my-5 sm:p-5">
+      <div className="relative z-10 mx-auto flex w-full max-w-3xl flex-col items-center justify-around border border-gray-800 bg-gray-800/50 text-center text-gray-200 shadow-xl shadow-black/20 transition sm:flex-row sm:rounded-lg sm:text-left">
+        <Image
+          src={require('../../../public/assets/ts-book.png')}
```

Comment:
why this over cloudinary?

Reply:
I just like the blurry placeholder but don't feel too strong about it

Reply:
it's using cloudinary now